pkgs/development/python-modules/ifcfg/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, fetchPypi
+, buildPythonApplication
+, python
+, iproute2
+}:
+
+buildPythonApplication rec{
+  pname = "ifcfg";
+  version = "0.23";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-YktaxUFpPM0+LOzDSMDogVTe1Kqt0lTLIxIS36r9vQI";
+  };
+
+  nativeBuildInputs = [
+    python
+  ];
```

Comment:
I don't think this is necessary.

Reply:
Yep, I think it was left over from when I was trying to run tests.